parasolr/query.py

```diff
@@ -161,67 +167,52 @@ def get_facets(self) -> Dict[str, int]:
 
     @staticmethod
     def _lookup_to_filter(key: str, value: Any) -> str:
-        """Convert keyword/value argument, with optional filters, into a
-        Solr query. Currently supports simple key-value pairs, searching for
-        empty/unset values, and __in filters of lists.
+        """Convert keyword/value argument, with optional lookups separated by
+        ``__``, including: in and exists. Field names should *NOT* include
+        double-underscores by convention.
 
             Returns: A propertly formatted Solr query string.
         """
-
-        # Flag if a null/empty search is being conducted
-        null_filter = False
-
-        # Base filter for all queries
-        _filter = ''
-
+        # NOTE: Should this go to a class or module constant?
+        ALL_VALUES = '[* TO *]'
```

Comment:
I think class constant for now - we don't know of another place we need it yet, right? And I would call it **ANY_VALUE** rather than **ALL**

Reply:
Although it might make sense for it to be the same place as the lookup separator... 

I'm fine with either, but it should be at class or module level and not buried in here.